Tests/ComposableArchitectureTests/BindableStoreTests.swift

```diff
@@ -80,4 +80,143 @@ final class BindableStoreTests: XCTestCase {
       }
     }
   }
+
+  func testTestStoreBindings() async {
+    struct LoginFeature: Reducer {
+      struct State: Equatable {
+        @BindingState var email = ""
+        public var isFormValid = false
+        public var isRequestInFlight = false
+        @BindingState var password = ""
+      }
+      enum Action: Equatable, BindableAction {
+        case binding(BindingAction<State>)
+        case loginButtonTapped
+      }
+      var body: some ReducerOf<Self> {
+        BindingReducer()
+        Reduce { state, action in
+          switch action {
+          case .binding:
+            state.isFormValid = !state.email.isEmpty && !state.password.isEmpty
+            return .none
+          case .loginButtonTapped:
+            state.isRequestInFlight = true
+            return .none  // TODO: Login request
```

Comment:
What do you think of "NB" here so that it doesn't seem like a TODO for us to fix?

```suggestion
            return .none  // NB: Login request
```